src/ctree-app/ctree-landing.html

```diff
@@ -346,22 +493,23 @@
       _dialogParamChanged(section) {
         if (section && section.length > 0) {
           this._scrollTo(this.$.content, this.$[section].offsetTop, 500);
-          this.section = section
+          this.section = section;
         }
       }
 
       _scrollTo(container, position, duration) {
-        var MILLIS_PER_FRAME = 16
-        var t = this
+        var MILLIS_PER_FRAME = 16;
+        var t = this;
         t._suppressInferSection = true;
-        var scrollStep = (position - container.scrollTop) / (duration / MILLIS_PER_FRAME);
+        var scrollStep =
+          (position - container.scrollTop) / (duration / MILLIS_PER_FRAME);
```

Comment:
I think it's cleaner on a single line.  I usually use 120 characters as a good guide width, although sometimes it's fine to go wider.